massa-db-worker/src/massa_db.rs

```diff
@@ -141,7 +137,7 @@ where
                                 }
                                 updates
                             }
-                            None => {
+                            _ => {
                                 return Err(MassaDBError::TimeError(String::from(
```

Comment:
Maybe we can rename the DB error here to something more meaningful? Like MassaDBError::CacheMissError or MassaDBError::CacheTooOldError ?

Reply:
Done